teuthology/provision/pelagos.py

```diff
@@ -0,0 +1,162 @@
+import json
+import logging
+import requests
+import socket
+import time
+import shutil
+import urllib2
+from urllib2 import urlopen, HTTPError
+import re
+
+import teuthology.orchestra
+from teuthology.config import config
+from teuthology.contextutil import safe_while
+from teuthology.exceptions import MaxWhileTries
+from teuthology import misc
+
+log = logging.getLogger(__name__)
+config_section = 'pelagos'
+
```

Comment:
Could we have at least some very basic docs (here in this file or in the teuthology rst docs) about the available config options in this section?